src/components/ActionsProvider.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
```

Comment:
Hi @icaromh, Could you please put the copyright holder and license inside this file?